set.archived.sh

```diff
@@ -30,7 +31,7 @@ echo "** Updating ${dataset_name}.analysis_archived table. **"
 #  "https://www.ebi.ac.uk/ena/portal/api/search" > "${output_dir}/analysis_archived.tsv"
-curl -X POST -H "Content-Type: application/x-www-form-urlencoded" -d 'result=analysis&query=study_accession%3D%22PRJEB43947%22%20OR%20study_accession%3D%22PRJEB45554%22%20OR%20study_accession%3D%22PRJEB45619%22%20OR%20study_accession%3D%22PRJEB55349%22%20OR%20study_accession%3D%22PRJEB55355%22%20OR%20study_accession%3D%22PRJEB55356%22%20OR%20study_accession%3D%22PRJEB57992%22%20OR%20study_accession%3D%22PRJEB57993%22%20OR%20study_accession%3D%22PRJEB57995%22%20OR%20study_accession%3D%22PRJEB59443%22%20OR%20study_accession%3D%22PRJEB59444%22%20OR%20study_accession%3D%22PRJEB59445%22%20OR%20study_accession%3D%22PRJEB61667%22%20OR%20study_accession%3D%22PRJEB61668%22%20OR%20study_accession%3D%22PRJEB61669%22%20OR%20study_accession%3D%22PRJEB70460%22%20OR%20study_accession%3D%22PRJEB70461%22%20OR%20study_accession%3D%22PRJEB70462%22&fields=sample_accession%2Crun_accession%2Canalysis_date%2Csubmitted_bytes%2Canalysis_type%2Cfirst_public&format=tsv&limit=0' \
+curl -X POST -H "Content-Type: application/x-www-form-urlencoded" -d 'result=analysis&query=study_accession%3D%22PRJEB43947%22%20OR%20study_accession%3D%22PRJEB45554%22%20OR%20study_accession%3D%22PRJEB45619%22%20OR%20study_accession%3D%22PRJEB55349%22%20OR%20study_accession%3D%22PRJEB55355%22%20OR%20study_accession%3D%22PRJEB55356%22%20OR%20study_accession%3D%22PRJEB57992%22%20OR%20study_accession%3D%22PRJEB57993%22%20OR%20study_accession%3D%22PRJEB57995%22%20OR%20study_accession%3D%22PRJEB59443%22%20OR%20study_accession%3D%22PRJEB59444%22%20OR%20study_accession%3D%22PRJEB59445%22%20OR%20study_accession%3D%22PRJEB61667%22%20OR%20study_accession%3D%22PRJEB61668%22%20OR%20study_accession%3D%22PRJEB61669%22%20OR%20study_accession%3D%22PRJEB70460%22%20OR%20study_accession%3D%22PRJEB70461%22%20OR%20study_accession%3D%22PRJEB70462%22%20OR%20study_accession%3D%22PRJEB74098%22%20OR%20study_accession%3D%22PRJEB74099%22%20OR%20study_accession%3D%22PRJEB74100%22&fields=sample_accession%2Crun_accession%2Canalysis_date%2Csubmitted_bytes%2Canalysis_type%2Cfirst_public&format=tsv&limit=0' \
```

Comment:
You should just use PRJEB74101 instead of the three child projects. The length of URL is limited.